build.gradle

```
@@ -60,7 +56,7 @@ repositories {
 dependencies {
     api platform('com.exactpro.th2:bom:3.0.0')
     implementation "com.exactpro.th2:common:3.13.6"
```

```suggestion
    implementation "com.exactpro.th2:common:3.14.0"
```
Maybe update to the last version and increase minor version of the current project